en/debug/reference-design.md

```diff
@@ -12,8 +12,8 @@ FMU generations 1-3 were designed as open hardware, while FMU generations 4 and
 
 * FMUv1: Development board \(STM32F407, 128 KB RAM, 1MB flash, [schematics](https://github.com/PX4/Hardware/tree/master/FMUv1)\) (no longer supported by PX4)
 * FMUv2: Pixhawk \(STM32F427, 168 MHz, 192 KB RAM, 1MB flash, [schematics](https://github.com/PX4/Hardware/tree/master/FMUv2)\)
-* FMUv3: Pixhawk variants with 2MB flash \(3DR Pixhawk 2 \(Solo\), Hex Pixhawk 2.1, Holybro Pixfalcon, 3DR Pixhawk Mini, STM32F427, 168 MHz, 192 KB RAM, 2 MB flash, [schematics](https://github.com/PX4/Hardware/tree/master/FMUv2)\)
-* FMUv4: Pixracer \(STM32F427, 168 MHz, 192 KB RAM, 2 MB flash, [pinout](https://docs.google.com/spreadsheets/d/1raRRouNsveQz8cj-EneWG6iW0dqGfRAifI91I2Sr5E0/edit#gid=1585075739)\)
-* FMUv4 PRO: Drotek Pixhawk 3 PRO \(STM32F467, 168 MHz, 384 KB RAM, 2 MB flash, [pinout](https://docs.google.com/spreadsheets/d/1raRRouNsveQz8cj-EneWG6iW0dqGfRAifI91I2Sr5E0/edit#gid=1585075739)\)
-* FMUv5: final name TBD \(STM32F7675, 200 MHz, 512 KB RAM, 2 MB flash, [pinout](https://docs.google.com/spreadsheets/d/1-n0__BYDedQrc_2NHqBenG1DNepAgnHpSGglke-QQwY/edit#gid=912976165)\)
+* FMUv3: Pixhawk variants with 2MB flash \(3DR Pixhawk 2 \(Solo\), Hex Pixhawk 2.1, Holybro Pixfalcon, 3DR Pixhawk Mini, STM32F427, 180 MHz, 256 KB RAM, 2 MB flash, [schematics](https://github.com/PX4/Hardware/tree/master/FMUv2)\)
+* FMUv4: Pixracer \(STM32F427, 180 MHz, 256 KB RAM, 2 MB flash, [pinout](https://docs.google.com/spreadsheets/d/1raRRouNsveQz8cj-EneWG6iW0dqGfRAifI91I2Sr5E0/edit#gid=1585075739)\) (Wrong Link)
```

Comment:
Why did you add 'Wrong Link' here?

Reply:
Looks like it only have 176-pin STM32F469IIT6 Signal pinout. But Pixracer is using STM32F427. Is that FMUv4-Mini the right one? Should we add some explanation at that cell.

Reply:
I see, you're right, Pixracer is using STM32F427. @davids5 can you help clarify that?

Reply:
The RAM change is correct.  The speed is incorrect. The CPU is capable of running at 180Mhz. The PX4 Firmware project uses USB and due to the F4's PLL limitations, that limits the the  CPU to running at 168 MHz.

Reply:
That is the the V4Pro pinout. But documents V2 and V4. I  was supposed be a comparison, with V2, V4 and V4 Pro I updated it with the SoC's part numbers for clarity